ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.18-nightly-x86.yaml

```diff
@@ -53,6 +53,40 @@ tests:
     - chain: openshift-qe-ingress-perf
     workflow: openshift-qe-installer-ibmcloud-ipi-ovn
   timeout: 5h0m0s
+- always_run: false
+  as: node-density-heavy-etcd-fips-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: gcp-qe
```

Comment:
you'll also want to update this profile to be ibm specific, should match whatever is above in this file